lib/devise/passkeys/controllers/passkeys_controller_concern.rb

```diff
@@ -90,7 +90,7 @@ def exclude_external_ids_for_registration
         end
 
         def user_details_for_registration
-          { id: resource.webauthn_id, name: resource.email }
+          { id: resource.webauthn_id, name: resource.send(resource_class.authentication_keys.first) }
```

Comment:
In the future maybe this may come from elsewhere, but I've tried to keep the current logic.

This shows up in Apple Passwords (and each other vendor app) and it might be cool to allow the user to pick something...
Just thinking out loud!
<img width="472" alt="Screenshot 2023-08-29 at 12 31 25 AM" src="https://github.com/ruby-passkeys/devise-passkeys/assets/1107956/1980dbe5-848d-4b6d-b2ad-12690997d31a">

Reply:
What if we expected a `resource.default_passkey_name` method, which the implementor could override as needed. `Devise::Models::PasskeyAuthenticatable` could have this default implementation:

```
def default_passkey_name
  self.class.authentication_keys.first
end
```

An important note is that the user is able to customize the name for their passkeys, since it's not really used for the key exchange. Hence the `default_` prefix here

Reply:
That is a good idea.
This will allow user to define any combination of user attribute to be what gets displayed in the device app (nickname, display_name formatted all sorts of way, etc..).